styles/donations.scss

```diff
@@ -146,6 +146,16 @@
     }
   }
 }
+.topProjectBadge{
+  position: absolute;
+  top: 30px;
+  right: 34px;
```

Comment:
It would be better to avoid absolute positioning in this case.

Reply:
It would be better to avoid absolute positioning in this case.

Reply:
@mohitb35  The top project badge will then be shifted to left for cases where back button is not visible. Is that okay?
![image](https://user-images.githubusercontent.com/72646230/234596697-f91fad1e-f095-4ffc-922f-cd143010a517.png)

Reply:
The `Top Project` badge should always be aligned right.

Reply:
Fixed